src/PIL/ImageFont.py

```diff
@@ -546,11 +546,20 @@ def truetype(font=None, size=10, index=0, encoding="", layout_engine=None):
     This function loads a font object from the given file or file-like
     object, and creates a font object for a font of the given size.
 
+    Note that Pillow uses FreeType to open font files. If you are opening many
```

Comment:
No need for "Note that":

```suggestion
   Pillow uses FreeType to open font files. If you are opening many
```